src/components/component-playground.jsx

```diff
@@ -181,12 +182,33 @@ module.exports = React.createClass({
   },
 
   _renderContentFrame: function() {
-    return <div ref="contentFrame" className={this._getContentFrameClasses()}>
-      <div ref="previewContainer" className={this._getPreviewClasses()}>
-        {this.loadChild('preview')}
+    var split = this.state.orientation == 'landscape' ?
+      'vertical' : 'horizontal';
+
+    // No need for a SplitPane if editor is not open
+    if (!this.props.editor) {
+      return (
+      <div ref="contentFrame" className={this._getContentFrameClasses()}>
+          <div ref="previewContainer" className={this._getPreviewClasses()}>
```

Comment:
You can extract this into a private render method and apply it here and in the else block.